terraform/modules/gclb/cert.tf

```diff
@@ -40,14 +31,25 @@ resource "google_certificate_manager_certificate_map" "map" {
 }
 
 resource "google_certificate_manager_certificate_map_entry" "map_entry" {
-  for_each = toset(var.new_domains)
+  for_each = var.domains
 
   name     = replace("certificatemapentry-${each.key}", ".", "-")
   map      = google_certificate_manager_certificate_map.map.name
+
   hostname = each.key
 
   certificates = [
     google_certificate_manager_certificate.cert[each.key].id
   ]
 }
 
+resource "google_certificate_manager_certificate_map_entry" "primary_map_entry" {
+  name     = replace("certificatemapentry-${var.primary_domain}-2", ".", "-")
```

Comment:
-2 ? this a blue/green thing?

Reply:
During our cert weirdness last week we added a primary cert map entry called `cgr-dev-2`, this just mirrors that into terraform.

If we want to we can remove the old non-primary one and promote this, I just didn't want to make prod diffs with this change.